reactioncommerce/reaction
# #3176 Fix search modal not closed on back/forward navigation

This PR fixes an issue that causes the search modal to stay open on navigating away from the current page using the back/forward button on the browser.

## How to Test
1. Add a product if none exist
2. View a product by clicking on a product
3. Click on Search
4. Press the Back button, the URL changes
5. Observe that the search modal is closed on leaving the current page

Closes #3176
